report_pre_thesis/src/sections/methodology.tex

```diff
@@ -1,8 +1,191 @@
+This author highlighted the extensive efforts invested in developing the ChemCam calibration dataset, suggesting a low probability of significant outlier presence.
+Given this context, introducing outlier detection for the ICA phase based on mere assumptions would be counterproductive, potentially compromising the integrity of our analysis.
+
+After examining the results in section \ref{sec:results}, we will discuss the implications of these design choices in section \ref{sec:discussion}.
+
+\subsection{Joint Approximate Diagonalization of Eigenmatrices (JADE) and Regression}
+After preprocessing the data, we are ready to perform ICA.
+This is done using the JADE algorithm, which is the same algorithm used by \citeauthor{cleggRecalibrationMarsScience2017}.
+This gives us eight independent components --- same as in the original pipeline.
+After performing ICA, we post-process by computing the correlation between the independent components and the wavelengths.
+Using this, we can identify which wavelengths are associated with which independent components by computing the maximum correlation for each wavelength.
+This gives us a matrix of ICA scores that we then use for regression.
+
+\begin{figure}
+	\centering
+	\includegraphics[width=0.45\textwidth]{images/ica_regression.png}
+	\caption{Regression in the ICA phase of our recreation of the pipeline.}
+	\label{fig:ica_regression}
+\end{figure}
+
+The regression process is illustrated in figure \ref{fig:ica_regression}.
+Since we are implementing this recreation in Python, a different programming language than the original pipeline, we cannot use the same libraries for regression.
+We use the \texttt{LinearRegression} class from the \texttt{scikit-learn} library\cite{scikit-learn}.
+However, \citeauthor{cleggRecalibrationMarsScience2017} mentions performing four more types of regression in addition to linear regression, namely parabolic, exponential, geometric and logsquare.
+For each oxide, they used the type of regression and normalization that produced the lowest root mean square (RMS) error.
+This can be seen in table \ref{tab:regression_types}, which is taken from \citeauthor{cleggRecalibrationMarsScience2017}.
+
+\begin{table}[h]
+\centering
+\begin{tabular*}{\columnwidth}{@{\extracolsep{\fill}}lccc}
+\toprule
+Element & Norm & Law type    & RMS \\ \midrule
+Si      & 1    & Logsqaure   & 6.7 \\
+Ti      & 3    & Geometric   & 0.6 \\
+Al      & 3    & Geometric   & 4.4 \\
+Fe      & 1    & Geometric   & 2.2 \\
+Mg      & 1    & Exponential & 3.0 \\
+Ca      & 1    & Parabolic   & 1.0 \\
+Na      & 3    & Parabolic   & 0.6 \\
+K       & 3    & Geometric   & 0.4 \\
+\bottomrule
+\end{tabular*}
+\caption{The summary of each ICA model characteristics from \citeauthor{cleggRecalibrationMarsScience2017}.}
+\label{tab:regression_types}
+\end{table}
+
+% Because \texttt{scikit-learn} does not support these types of regression, we instead opted to transform the data to fit the linear regression model.
```

Comment:
TODO?

Reply:
```suggestion
% TODO: Update/rewrite this to reflect the new curve-fit implementation.
% Because \texttt{scikit-learn} does not support these types of regression, we instead opted to transform the data to fit the linear regression model.
```